tests/testthat/test_validate_factory.R

```diff
@@ -16,7 +16,9 @@ test_that("A new_factory is valid", {
     length(test$errors) == 0L && length(test$warnings) == 0L
   }
   expect_true(no_probs(validate_factory(x)))
-  update_reports(quiet = TRUE, factory = x)
+  test_report <- grep(list_reports(x), pattern = "contacts", value = TRUE)[1]
```

Comment:
For consistency, would `list_report(pattern = "contacts")` work here as well?

Reply:
ahh right, yes